reference/conanfile/tools/gnu/autotoolstoolchain.rst

```diff
@@ -232,3 +232,46 @@ The ``environment()`` method returns an :ref:`Environment<conan_tools_env_enviro
             env = at.environment()
             env.define("FOO", "BAR")
             at.generate(env)
+
+
+Customizing the xxxxxx_args attributes
```

Comment:
I think if would be nicer if you use the real names in the title instead of xxxxx_args